ledger-tool/src/main.rs

```diff
@@ -1066,16 +1066,15 @@ fn load_bank_forks(
     };
 
     if let Some(halt_slot) = process_options.halt_at_slot {
```

Comment:
Ideally we'd just check halt_slot != 0 here so we can make a comment about the special case, but I don't think rust allows chained conditionals with if let yet. An alternative is something like:
```
match process_options.halt_at_slot {
    // Sentinel value of `Some(0)` lets us skip additional checks and will not replay any transactions.
    Some(halt_slot) if halt_slot != 0 => { /*...*/ },
    _ => {},
}
```

Reply:
Also happy if we just add a comment above 1068 with your current changes.

Reply:
yeah. good idea.